README.md

```diff
@@ -15,16 +15,34 @@ Versions prior to 3.1.68 are vulnerable to a reflected XSS attack. Please update
 
 ## Installation
 
+This package depends on the [Shopify Node Library](https://github.com/Shopify/shopify-node-api) being included in your app, so you should install both of them:
```

Comment:
*should* install it? Isnt it already installed via dependency?

Reply:
So this is a bit weird, and I haven't had a chance to test this before we release `koa-shopify-auth` - if you use a local import you're not able to import the dependency directly, but I fully expect it to work with an actual package import. I didn't want to state that here until I had a chance to verify it.